master/buildbot/status/web/console.py

```diff
@@ -316,26 +318,26 @@ def getAllBuildsForRevision(self, status, request, codebase, lastRevision,
     #
     # Display functions
     #
-    def displayCategories(self, builderList, debugInfo):
-        """Display the top category line."""
+    def displayTags(self, builderList, debugInfo):
+        """Display the top tags line."""
 
         count = 0
-        for category in builderList:
-            count += len(builderList[category])
+        for tag in builderList:
+            count += len(builderList[tag])
 
-        categories = sorted(builderList.keys())
+        tags = sorted(builderList.keys())
 
         cs = []
 
-        for category in categories:
+        for tag in tags:
```

Comment:
Yes, this would definitely look better as a list-comp ... In general, I tried to not rewrite code that wasnt actually changing behavior. I just have been wrong too many times by "obviously the same thing" kinds of changes :) Since this code is dead-end in eight, I'm tempted to just let it be unless you have a strong objection.